src/Numerical/Spectral/MeanValue.hpp

```diff
@@ -0,0 +1,41 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines function MeanValue.
+
+#pragma once
+
+#include "Domain/Side.hpp"
+#include "Numerical/Spectral/DefiniteIntegral.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+
+/*! \ingroup Functors
```

Comment:
I think this has to be on a newline for it to render?

Reply:
We also don't have such a group